### PR TITLE
[SMALLFIX] Update Alluxio on GCE docs

### PR DESCRIPTION
**Pull request by gpang**
@aaudiber

### CODE REVIEW

File: docs/en/Running-Alluxio-on-GCE.md [hunk 1]
```diff
@@ -62,13 +69,14 @@ Copy `deploy/vagrant/conf/gce.yml.template` to `deploy/vagrant/conf/gce.yml` by:
 
 In the configuration file `deploy/vagrant/conf/gce.yml`, set the project id, service account, location to JSON key and ssh username you've just created.
 
-For GCE, the default underfs is Google Cloud Storage (GCS). You need to sign into your
-[Google Cloud console](https://console.cloud.google.com), create a GCS bucket and write
-the bucket's name to the field `GCS:Bucket` in `conf/ufs.yml`. To use other under storage systems,
-configure the field `Type` and the corresponding configurations in `conf/ufs.yml`.
+For GCE, the default underfs is Google Cloud Storage (GCS). Visit the
+[Storage](https://console.cloud.google.com/storage/)  of the Google Cloud console, create a GCS
```
Comment (aaudiber) [inline docs/en/Running-Alluxio-on-GCE.md]: nit: extra space before `of`

Comment (aaudiber) [reply docs/en/Running-Alluxio-on-GCE.md]: should it say `Storage page` or `Storage tab`?

File: docs/en/Running-Alluxio-on-GCE.md [hunk 2]
```diff
@@ -62,13 +69,14 @@ Copy `deploy/vagrant/conf/gce.yml.template` to `deploy/vagrant/conf/gce.yml` by:
...
+bucket, and fill in the bucket's name to the field `GCS:Bucket` in `conf/ufs.yml`. To use other
```
Comment (aaudiber) [inline docs/en/Running-Alluxio-on-GCE.md]: grammar: "fill in ... with" or "set ... to"

Comment (gpang) [reply docs/en/Running-Alluxio-on-GCE.md]: Done

File: docs/en/Running-Alluxio-on-GCE.md (hunk 1 above)
Comment (gpang) [reply docs/en/Running-Alluxio-on-GCE.md]: Done

**Pull request merged**